### PR TITLE
[Docker] Change Dockerfile and document

### PR DESCRIPTION
**Pull request by HuangFru**
## Why are the changes needed?

Arctic update project version to 0.3.1, so Dockerfile and some document need to update.

## Brief change log

  - *Change Arctic Meta Service image's Dockerfile*
  - *Change Flink image's Dockerfile*
  - *add some tips in document*

## How was this patch tested?
- [ ] Add some test cases that check the changes thoroughly including negative and positive cases if possible

- [ ] Add screenshots for manual tests if appropriate

- [x] Run test locally before making a pull request

## Documentation

  - Does this pull request introduces a new feature? (no)
  - If yes, how is the feature documented? (docs)

### CODE REVIEW

File: docker/flink/Dockerfile [hunk 1]
```diff
@@ -3,21 +3,27 @@ FROM openjdk:8u332-jdk
 
 RUN apt update \
     && apt-get install -y netcat \
-    && apt-get install -y vim
+    && apt-get install -y vim \
+    && apt-get install -y net-tools \
+    && apt-get install -y telnet
 
 WORKDIR /usr/local/arctic_flink
+#需自行下载flink-1.12.2-bin-scala_2.11.tgz放在Dockerfile同级目录下
```
Comment (zhoujinsong) [inline docker/flink/Dockerfile:11]: Change comment to english.

File: docker/flink/Dockerfile [hunk 2]
```diff
@@ -3,21 +3,27 @@ FROM openjdk:8u332-jdk
...
+RUN tar -zxvf flink-${FLINK_VERSION}-bin-scala_${SCALA_VERSION}.tgz \
     && wget https://repo1.maven.org/maven2/org/apache/flink/flink-shaded-hadoop-2-uber/${HADOOP_VERSION}-10.0/flink-shaded-hadoop-2-uber-${HADOOP_VERSION}-10.0.jar \
-    && wget https://github.com/NetEase/arctic/releases/download/v0.3.0-rc1/arctic-flink-runtime-1.12-0.3.0.jar \
+    && wget https://github.com/NetEase/arctic/releases/download/v0.3.1-rc1/arctic-flink-runtime-1.12-0.3.1.jar \
     && cd flink-${FLINK_VERSION} \
     && cp examples/table/ChangelogSocketExample.jar lib \
-    && cp ../arctic-flink-runtime-1.12-0.3.0.jar lib \
+    && cp ../arctic-flink-runtime-1.12-0.3.1.jar lib \
     && cp ../flink-shaded-hadoop-2-uber-${HADOOP_VERSION}-10.0.jar lib
+#需自行下载hive-exec-2.1.1.jar放在Dockerfile同级目录下
```
Comment (zhoujinsong) [inline docker/flink/Dockerfile:24]: Change comment to english.

**Review by zhoujinsong — CHANGES_REQUESTED** [review]
@HuangFru  Recommend comment in english.

File: docker/flink/Dockerfile (hunk 1 above)
Comment (HuangFru) [reply docker/flink/Dockerfile]: done

File: docker/flink/Dockerfile (hunk 2 above)
Comment (HuangFru) [reply docker/flink/Dockerfile]: done

**Review by zhoujinsong — APPROVED** [review]
LGTM.

**Pull request merged**